homeassistant/components/filter/sensor.py

```diff
@@ -497,7 +497,7 @@ def __init__(
         :param upper_bound: band upper bound
         :param lower_bound: band lower bound
         """
-        super().__init__(FILTER_NAME_RANGE, precision=precision, entity=entity)
+        super().__init__(FILTER_NAME_RANGE, precision=None, entity=entity)
```

Comment:
```suggestion
        super().__init__(FILTER_NAME_RANGE, precision=precision, entity=entity)
```

This is unintentionally ignoring the `precision` argument being passed in.